src/appConfig.yml

```diff
@@ -4,24 +4,35 @@ httpServer:
   host: "localhost"
   port: 8080
 
-# Database config [MySQL only]
+# Database config
+# Currently supports MySQL and SQLite db drivers
 database:
   entity:
-    dbName: "roldb"
-    username: "root"
-    password: "67Edh68Tyt69"    # pragma: allowlist secret
-    protocol: "tcp"
-    hostname: "localhost"
-    port: "3306"
-    parameters: "?charset=utf8mb4&parseTime=True&loc=Local"
+    driver: "mysql"
```

Comment:
let's set SQLite the default one for now

Reply:
Okay